src/qt/locale/bitcoin_nl.ts

```diff
@@ -4421,75 +4421,75 @@ p, li { white-space: pre-wrap; }
     </message>
     <message>
         <source>Amaze</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Verbazend</translation>
     </message>
     <message>
         <source>Default</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Standaard</translation>
     </message>
     <message>
         <source>Many generous</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Heel genereus</translation>
     </message>
     <message>
         <source>Minimum</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Minimum</translation>
     </message>
     <message>
         <source>More</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Meer</translation>
     </message>
     <message>
         <source>Receive and display P2P network alerts (default: %u)</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">P2P-netwerkwaarschuwingen ontvangen en weergeven (default: %u)</translation>
     </message>
     <message>
         <source>Such expensive</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Zo duur</translation>
     </message>
     <message>
         <source>Wow</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Wow</translation>
     </message>
     <message>
         <source>-discardthreshold is set very high! This is the output amount that the wallet will discard (to fee) if it is smaller than this setting.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">-afgooidrempel is erg hoog! Dit is het uitvoerbedrag dat de portemonnee zal weggooien (tegen vergoeding) als het kleiner is dan deze instelling.</translation>
     </message>
     <message>
         <source>Amount under which a transaction output is considered non-standard and will not be accepted or relayed, in %s (default: %s)</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Bedrag waaronder een transactie-output als niet-standaard wordt beschouwd en niet wordt geaccepteerd of doorgegeven, in %s (default: %s)</translation>
     </message>
     <message>
         <source>Invalid amount for -discardthreshold=&lt;amount&gt;: &apos;%s&apos; (must be at least the dust limit of %s to prevent stuck transactions)</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">bedrag voor -drempelbedrag weggooien=&lt;amount&gt;: &apos;%s&apos; (moet minimaal de stoflimiet zijn van om vastgelopen transacties te voorkomen)</translation>
```

Comment:
- `bedrag voor -drempelbedrag weggooien` => `Ongeldig bedrag voor -discardthreshold` - missed "invalid" and shouldn't translate parameter name.

Reply:
not sure if `-discardthreshold=&lt;amount&gt;: &apos;%s&apos;` should stick onto each other.

Reply:
it should, it translates to `-discardthreshold=<amount>: '%s'`